src/Http/Controllers/FilesController.php

```diff
@@ -93,6 +93,13 @@ private function files()
                     $data['fileMeta'][$key] = $file[$column] ?? null;
                 }
 
+                if ($this->tool->relationship === 'HasMany' || $this->tool->relationship === 'HasMany') {
+                    $base = config('nova.url') . config('nova.path');
+                    $url = url("$base/resources/{$this->tool->attribute}/{$file['id']}");
```

Comment:
Better to return `$data['modelId']` then compose the url on the front-end

Reply:
I'm not sure how do it in the frontend!!!